schema/me/conversation/index.js

```diff
@@ -179,18 +184,42 @@ export const ConversationFields = {
   is_last_message_to_user: {
     type: GraphQLBoolean,
     description: "True if user/conversation initiator is a recipient.",
-    resolve: ({ _embedded, from_email }) => {
-      const lastMessageFromEmail = get(_embedded, "last_message.from_email_address")
-      return from_email !== lastMessageFromEmail
-    },
+    resolve: conversation => isLastMessageToUser(conversation),
   },
 
   // If the user is a recipient of the last message, return their timestamped
   // 'read' event, otherwise null.
   last_message_open: {
     type: GraphQLString,
     description: "Timestamp if the user opened the last message, null in all other cases",
-    resolve: conversation => null, // eslint-disable-line no-unused-vars
+    resolve: (conversation, options, request, { rootValue: { accessToken } }) => {
+      if (!isLastMessageToUser(conversation)) {
+        return null
+      }
+      const radiationMessageId = get(conversation, "_embedded.last_message.radiation_message_id")
+      const impulseParams = {
+        conversation_id: conversation.id,
+        radiation_message_id: radiationMessageId,
+        "expand[]": "deliveries",
+      }
+      return gravity.with(accessToken, { method: "POST" })("me/token", {
+        client_application_id: IMPULSE_APPLICATION_ID,
+      }).then(data => {
```

Comment:
@mzikherman This might make for a prime candidate for a first authenticated loader so that we only make this request once per GraphQL query.